app/portal/chapters/page.js

```diff
@@ -58,14 +58,17 @@ export default function Page() {
         <div className="header-row">
           <h1>Chapters</h1>
           {/* TODO: This button should only be visible to super admins */}
-          <IconButton
+          {
+            current_user && current_user.role === 'admin' && current_user.chapter === 'Code Your Dreams' ? 
```

Comment:
I wonder if we should avoid using a string for the org name and use the chapter id instead? Perhaps we can add a parameter in `.env` that sets the ID for the CYD org to make sure we don’t have issues with determining what org is what (for example, what happens if a user changes the text name of “Code Your Dreams” by accident… a scary thought!)

Reply:
"parameter" meaning environment variable — I couldn't come up with the right word at the time! 😅

Reply:
I think you raise a good point here - and the fix is pretty easy to implement! Our database already "knows" which id is linked to which Chapter, and `current_user` also has a `chapter_id` we can tap into!

I'll update these lines to `current_user && current_user.role === 'admin' && current_user.chapter_id === 0 ?`, if that sounds like a solid fix!

Reply:
Technically, we could rely on integers rather than strings for user role here too, but it would require slightly bigger changes to the back-end - would need to introduce a new intermediary table that contains role names and links 'em to IDs

Reply:
As discussed in Slack, it sounds like a plan—go ahead and do what you need to do on the backend and then we can circle around to pushing this forward when you're ready. Thanks again!